source/_patterns/02-molecules/card/_card.scss

```diff
@@ -1,5 +1,9 @@
 @import '../../00-base/colors';
```

Comment:
I'd like to see _card.scss require button.scss explicitly, if the card uses a button.